store.go

```diff
@@ -523,6 +539,12 @@ func (s *store) load() error {
 	if err != nil {
 		return err
 	}
+	rois, err := s.ROImageStores()
+	if err != nil {
+		return err
+	}
+	s.roImageStores = rois
```

Comment:
This assignment looks redundant, since `ROImageStores()` is already making sure `s.roImageStores` is set.